### PR TITLE
add a zip cmd

### PR DESCRIPTION
**Pull request by ihaolin**
zip exclude some patterns

### CODE REVIEW

File: pages/common/zip.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Remove unwanted files from an existing zip file:
 
 `zip -d {{compressed.zip}} "{{foo/*.tmp}}"`
+
+- Remove unwanted pattern files from target dirs:
```
Comment (igorshubovych) [inline pages/common/zip.md]: This can be even more simplified:

```md
- Remove unwanted files from target dirs:
```

Comment (ihaolin) [inline pages/common/zip.md]: I agree with you.

Comment (igorshubovych) [inline pages/common/zip.md]: So you can just another commit to branch `ihaolin:patch-1`

Comment (ihaolin) [inline pages/common/zip.md]: Ok, I have commit it.

**Comment by igorshubovych** [comment]
Thank you for contribution!

**Pull request merged**